File: mod1/practical-example/imc-computation-ws/imc.js
//Script example to learn how to create a web page
// that uses js+css+html
//This page will calculate the IMC of the user
//Created in 2020-07-14 by Aline Guerreiro

//function to compute imc (bmi = body mass index)
function computeIMC(){
    var interpretResult = ""
    var validResults = ""

    var weight = parseFloat(document.getElementById("weight").value)
    var height = parseFloat(document.getElementById("height").value)
  
    if(weight !== 0 && height !== 0){
        var imc = weight/(height*height)

        if(imc >= 40){
            interpretResult = "severe obesity :("
        } else if((imc >= 30)&&(imc <= 39.9)){
            interpretResult = "obesity :("
        } else if((imc >= 25)&&(imc <= 29.9)){
            interpretResult = "overweight :("
        } else if((imc >= 18.5)&&(imc <= 24.9)){
            interpretResult = "normal IMC. Congrats!"
        } else {
            interpretResult = "less than normal IMC :("
        }
    
    document.getElementById("result").innerHTML= "Your IMC is: " + imc.toFixed(2)
    document.getElementById("interpretResult").innerHTML = "That means you are with: " + interpretResult
    
    } else {
        validResults = "Please, input no empty information."
        document.getElementById("validResult").innerHTML = validResults
    } 
}

function cleanResults(){
    document.getElementById("weight").value = 0
    document.getElementById("height").value = 0
    document.getElementById("result").innerHTML = ""
    document.getElementById("interpretResult").innerHTML = "" 
}

File: mod4/practical-examples/imc.py
#Script of practical-example from module 4 in Python
#Created in 2020-08-29 by Aline Guerreiro

weight = float(input("Enter your weight: "))
height = float(input("Enter your height: "))

imc = weight / (height**2)

print("Your IMC is : {}".format(imc))

if imc < 17:
    print("You are low weight")
elif (imc >=17)and(imc<18.5):
    print("You are low weight")
elif (imc >=18.5)and(imc<25):
    print("You are normal weight")
elif (imc >=25)and(imc<30):
    print("You are high weight")
elif (imc >=30)and(imc<35):
    print("You are level 1 obesidity")
elif (imc >=35)and(imc<40):
    print("You are level 2 obesidity")
elif (imc > 40):
    print("You are morbid obesidity")